spaceship/spec/spec_helper.rb

```diff
@@ -30,10 +32,6 @@ module SpecHelper
 ENV["DELIVER_PASSWORD"] = "so_secret"
 ENV.delete("FASTLANE_USER") # just in case the dev env has it
 
-unless ENV["DEBUG"]
-  $stdout = File.open("/tmp/spaceship_tests", "w")
-end
```

Comment:
Yes, the results print in Circle, but the amount is not all that much.  It would be overwhelming if the Faraday logging was in there, but just the pass/fail test results are not that much.